.travis.yml

```diff
@@ -6,6 +6,22 @@
 sudo: false
 language: python
 install: pip install tox
-script: tox
-python:
-  - 3.5
+script: tox -e $TOX_ENV
+matrix:
+  include:
+    - python: 2.6
```

Comment:
If you make these `TOXENV` you only need do `tox` as the script because `tox` will pick it up for you.